### PR TITLE
RestaurantList (bottom portion for Landing page view) completed with unit test

### PR DESCRIPTION
**Pull request by valerieyang00**
** Please skip reviewing LandingMap for this PR, it is work in progress, will be in next PR **

To review:
- RestaurantList / Test (todo: more information will be displayed as Restaurant Model gets updated with discussed additional fields)
- Added type to MockReviews in Restaurant.test.tsx as suggested in previous PR approved.

RowsPerPage-5 
restaurantlist-first

RowsPerPage-10 1/2
restaurantlist-10

RowsPerPage-10 2/2
restaurantlist-last

### CODE REVIEW

**Comment by moyara** [comment]
could you exclude LandingMap from the PR if you don't want it to be reviewed? it might be merged and missed in the future PR.

File: src/components/partials/RestaurantList.test.tsx [hunk 1]
```diff
@@ -0,0 +1,197 @@
+import { fireEvent, render, screen, within } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import TestProvider from '../partials/TestProvider'
+import RestaurantList from './RestaurantList'
+
+const mockRestaurants = [
```
Comment (moyara) [inline src/components/partials/RestaurantList.test.tsx]: this is also missing type

**Review by moyara — APPROVED** [review]
minor nits. looks good.

**Comment by valerieyang00** [comment]
> could you exclude LandingMap from the PR if you don't want it to be reviewed? it might be merged and missed in the future PR.

This was completed

File: src/components/partials/RestaurantList.test.tsx (hunk 1 above)
Comment (valerieyang00) [reply src/components/partials/RestaurantList.test.tsx]: Added and pushed

**Pull request merged**